pages/Form.tsx

```diff
@@ -8,8 +7,8 @@ interface FormProps {
   onFormSubmit: (campaign: Campaign) => void;
 }
 
-const Form = ({ onFormSubmit, games, languages }: FormProps) => {
-  const [visible, setVisible] = useState<boolean>(false);
+export default function Form({ onFormSubmit, games, languages }: FormProps) {
+  const [visible, setVisible] = useState(false);
   const [campaign, setCampaign] = useState<Campaign>({
```

Comment:
This is not page should not be here. https://nextjs.org/docs/pages/building-your-application/routing/pages-and-layouts

Reply:
done